default-config.yml

```diff
@@ -0,0 +1,106 @@
+
+  # Token to connect to the Twitch IRC. You can get one at
+  # https://twitchapps.com/tmi/. You probably don't want to use your own
+  # personal account, but rather a testing or dev account.
+  oauth_token: <token not set>
+
+  # Channel that the bot connects to. Unless you are deploying, this should
+  # _never_ be the live channel (i.e. "baalorlord"). Doing so will make the
+  # account connected to the OAuth token above to respond to bot commands
+  # (amongst other things).
+  # You probably want it to be your own private channel (i.e. your username) if
+  # you're testing things out.
+  channel: <channel not set>
+
+discord:
+  # If the Discord part of the bot is enabled or not. At least one of the Twitch
+  # bot or the Discord bot need to be enabled for the server to function.
+  enabled: true
+
+  # TODO: Comment on how to obtain one of these.
+  oauth_token: <token not set>
+
+  # Invite links to the servers. Should be the full https:// URLs.
+  invite_links:
+    # Both of these are not rickrolls.
+    main: https://www.youtube.com/watch?v=dQw4w9WgXcQ
+    dev: https://www.youtube.com/watch?v=QZ1U2jvJgUg
+
+  # The server ID the bot is working on. This acts as a guard so it doesn't send
+  # messages to other servers.
+  server_id: <channel id not set>
+
+  # Which user role is considered a moderator. This should be the numberical ID
+  # of the role, and the default of 0 will make it error out.
+  moderator_role: 0
+
+youtube:
+  # The API key to make Youtube calls.
+  api_key: <api key not set>
+
+  # The channel ID used when searching
+  channel_id: <channel id not set>
+
+  # If no video ID is found via search, use this instead. (most likely because
+  # you're in a development context)
+  default_video: "fei2frQWPxs"
+
+  # Amount of seconds we wait between making API calls to Youtube. Should
+  # probably never be lower than 3600.
+  cache_timeout: 3600
+
+baalorbot:
+  # What prefix the bot listens to
+  prefix: "!"
+
+  # These correspond to the permission groups for the bot.
+  owners: []
+  editors: []
+
+  # Commands that run automatically over certain intervals.
+  timers:
+    globals:
+      interval: 300
+      commands: []
+    sponsored:
+      interval: 300
+      commands: []
+
+server:
+  # If this is true, we enable debug logging.
+  debug: false
+
+  # Secret key used when making calls from the client. Needs to be the same on
+  # both the client side and the server side. The default is not set for security reasons.
+  secret: ""
+
+  # The URL used when rendering links and redirects.
+  url: http://localhost:8080
+
+  # Amount of spaces when dumping JSON files.
+  json_indent: 2
+
+  # This email is visible in the footer of every page. For development contexts,
+  # this can be left empty.
+  business_email: <email not set>
+
+  # Websocket and webhook setup. TODO: Document.
+  websocket_client:
+    id: "889952b3-7b44-148e-9323-f38ccabb2744"
+    secret: "2f55d379-a27c-1101-dbfe-25c70e44f330"
```

Comment:
Is it fine to have these in the repo?

Reply:
These are dummy UUIDs, they're invalid. Though it could be made more obvious

Reply:
That's the TODO part, since I don't know that much about this part of the code.

Reply:
Fixed in a new commit.